Update compilation info for jazzy (#309)
ros-controls/control.ros.org: doc/getting_started/getting_started.rst

Building from Source
---------------------------

To receive the latest features and bug fixes or if you want to contribute to the framework, you can build the framework from source.

You can choose between the following options:

>    * Stable version: These branches will be released in the future in the respective ROS 2 distribution binaries.

Reviewer: ```suggestion
   * Stable version: The changes in these branches will be tagged and released to the ROS 2 distribution binaries.
```
